index.html

```diff
@@ -353,14 +353,22 @@ <h1>Important Terms</h1>
 			<dd>
 			  <p>Usable by users in ways they can control. References in this document relate to <cite><a href="https://www.w3.org/TR/WCAG21/#operable"><abbr title="Web Content Accessibility Guidelines">WCAG</abbr> 2.1 Principle 2: Content must be operable</a></cite> [[WCAG21]]. See <a>Keyboard Accessible</a>.</p>
 			</dd>
-			<dt><dfn data-export="" data-dfn-for="ARIA" data-local-lt="owned|owned element's">Owned Element</dfn></dt>
+			<dt><dfn data-export="" data-lt="owned child|child element|child|children|child elements">Owned Child</dfn></dt>
+			<dd>
+				<p>An 'owned child' is a direct <abbr title="Document Object Model">DOM</abbr> child of the <a>element</a>, a descendant of the <a>element</a> with only <rref>generic</rref> or <rref>presentation</rref> elements intervening, or a child specified via an <pref>aria-owns</pref> relationship to the element.</p>
```

Comment:
This is a good question, but I don't think this PR meaningfully changes the question or the answer. Would you be up for opening a separate issue against the spec for clarifying DOM parent/child relationship mappings when `aria-owns` is also present?

My personal assumption would be a replacement of the parent relationship rather than an addition, but I'm curious what other people think. A quick test shows Chrome and Firefox on Windows move the owned element in the accessibility tree, but do not duplicate it.

Reply:
https://github.com/w3c/aria/issues/1817

Reply:
IMO, "owned child" is the wrong term for the reasons outlined in the previous comment. 

I concur with Sarah that the owned element should be be moved in the AX tree, not copied. Note that this is a more recent change than the original understanding of aria-owns in ARIA 1.0. Several of those earlier meetings referenced author based aria-owns changes affecting the tree as the Ghostbusters meme: "Don't cross the streams… It would be bad."

However, most of the implementation challenges have been resolved since then, and I think all implementors are now in agreement that "move in the AX tree" should replace the "keep in place but add a cross-ref pointer" which was the original understanding. Those changes are are only partially implemented though.